core/src/main/java/org/geysermc/geyser/event/type/GeyserBedrockPingEventImpl.java

```diff
@@ -42,33 +44,33 @@ public GeyserBedrockPingEventImpl(BedrockPong pong, InetSocketAddress address) {
 
     @Override
     public void setMotd1(@Nonnull String motd1) {
-        if (motd1 == null) throw new NullPointerException("Motd1 cannot be null");
+        Objects.requireNonNull(motd1, "Motd1 cannot be null");
         pong.setMotd(motd1);
     }
 
     @Override
     public void setMotd2(@Nonnull String motd2) {
-        if (motd2 == null) throw new NullPointerException("Motd2 cannot be null");
+        Objects.requireNonNull(motd2, "Motd2 cannot be null");
         pong.setSubMotd(motd2);
     }
 
     @Override
     public void setPlayerCount(int count) {
-        pong.setPlayerCount(count);
+        if (count >= 0) pong.setPlayerCount(count);
     }
 
     @Override
     public void setMaxPlayerCount(int max) {
-        pong.setMaximumPlayerCount(max);
+        if (max > 0) pong.setMaximumPlayerCount(max);
```

Comment:
Sorry I should have specified - throw an illegal argument exception instead with a message, if the number is out of bounds

Reply:
I'll assume I add it to both playercount setters right?